README.markdown

```diff
@@ -1,68 +1,104 @@
-# DITA Open Toolkit [![Build Status][1]](http://travis-ci.org/dita-ot/dita-ot) [![Slack][7]](http://slack.dita-ot.org/)
+# DITA Open Toolkit [![Build]](http://travis-ci.org/dita-ot/dita-ot) [![Slack]](http://slack.dita-ot.org/)
 
-The _DITA Open Toolkit_, or _DITA-OT_ for short, is an open-source publishing engine for XML content authored in the _Darwin Information Typing Architecture_. 
+The _DITA Open Toolkit_, or _DITA-OT_ for short, is an open-source publishing engine for XML content authored in the _Darwin Information Typing Architecture_.
 
-See [dita-ot.org][2] for documentation, information about releases, and download packages. 
+Visit the project website at [dita-ot.org][site] for documentation, information about releases, and [download packages][dist].  
+For information on additional DITA and DITA-OT resources, see [SUPPORT].
 
-For information on additional DITA and DITA-OT resources, see [SUPPORT][8]. 
+- [Prerequisites](#prerequisites)
+- [Installing](#installing)
+- [Building output](#building-output)
+- [For developers](#for-developers)
+- [License](#license)
 
 ## Prerequisites
 
 To build and use DITA-OT, you’ll need:
 
-* Java Development Kit 8 or newer
+- Java Development Kit 8 or newer
```

Comment:
This could be fixed in this PR. So to build DITA-OT you need JDK (Java Development Kit), but to use it, you can just use JRE (Java Runtime Environment.)

Reply:
Sure, I can clarify that while we're at it.